contents/testing/writing-testable-code.md

```diff
@@ -0,0 +1,110 @@
+# Writing Testable Code
+
+Below is a collection of 10 rules that you can follow to write testable code.
```

Comment:
Just my opinion, saying it this way makes me think of those top 10 click bait articles. Overall, the article seems to assume that the reader has knowledge of testing. Is that intended?